app/src/main/res/layout/small_info_card_fragment.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/infocardLinearLayout"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical">
+
+    <TextView
+        android:id="@+id/info_card_title"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:drawableTop="@drawable/ic_line_grey"
+        android:gravity="center"
+        android:paddingLeft="10dp"
+        android:paddingTop="5dp"
+        android:paddingRight="10dp"
+        android:paddingBottom="10dp"
+        android:text="TextView"
+        android:textAllCaps="false"
+        android:textColor="@color/colorAppTheme"
+        android:textSize="20sp" />
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="30sp"
+        android:layout_marginEnd="30sp"
+        android:layout_marginBottom="35sp"
+        android:orientation="horizontal">
+
+        <Button
+            style="@style/InfoCardButton"
+            android:id="@+id/directions"
+            android:layout_marginEnd="15sp"
+            android:drawableLeft="@drawable/ic_directions_black_24dp"
+            android:text="Directions"
```

Comment:
Every text should go in strings.xml, in this case, Directions is written the same way in french but if it was not then we wouldn't be able to get it displayed translated. I am currently going through the whole project and adding the texts to strings.xml and french version of strings.xml. but for new additions, we should already put them there.

Reply:
done